src/main/java/uk/ac/ebi/ampt2d/registry/entities/User.java

```diff
@@ -24,7 +24,7 @@
 import javax.persistence.Enumerated;
 import javax.persistence.Id;
 
-@Entity
+@Entity(name = "users")
```

Comment:
The rest of the table names are in singular, this is inconsistent.

Reply:
With solution to postgreSQL reserved word "User" new name is RegistryUser. 
However Spring by default uses org.springframework.boot.orm.jpa.SpringNamingStrategy which splits camel case names with underscore. 

Now either we need to force setting in application.properties to make it REGISTRYUSER (Single word) as 
spring.jpa.hibernate.naming.physical-strategy=org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
This applied to future class names aswell

or if the convention permits use REGISTRY_USER.

Reply:
URL retained as /users